test/spec/modules/onetagBidAdapter_spec.js

```diff
@@ -81,7 +81,7 @@ describe('onetag', function () {
         }
       });
     } catch (e) {
-      console.log('Error while parsing');
+      console.log('onetagBidAdapter tests: Error while parsing');  // eslint-disable-line
```

Comment:
any reason to keep these these?

Reply:
I thought at first they would be good to keep for the adapter's tests, but in retrospect they're not too helpful.  I pushed an update to remove them.